### ISSUE TITLE
Multiedit makes strange changes

### ISSUE THREAD

**Issue by urskaufmann** [QGEP/QGEP#445]
if I switch from on of the other tabs to the wasterwater_structure-tab in vw_qgep_reach or vw_qgep_wastewater_structure in multiedit-mode, then there is the question, if I want to save the changes or not. But I have made no changes, I have just klicked on the tabs. Its probably the last_modification-field? But this field is not in the form.
![multiedit-error](https://user-images.githubusercontent.com/1257403/44411741-823c3b00-a567-11e8-998d-c98d4ff2c9b2.PNG)

**Comment by urskaufmann** [comment]
If I then click the black/white cross to close the orange message and afterwoods click the editing-tool to stop editing, I get a QGIS-crash...

**Comment by sjib** [comment]
@urskaufmann Still an issue?

**Comment by urskaufmann** [comment]
unfortunately it is. Tests with QGIS 3.4.8 and 3.6.3.
Its probably not the last modification-field as I mentioned in the first post

![multiedit2](https://user-images.githubusercontent.com/1257403/59033091-275ea080-8868-11e9-9096-23ba211e8063.PNG)
The yellow message appears when I change to the wastewater structure tab. If I click on "anwenden" , I get this:
![multiedit3](https://user-images.githubusercontent.com/1257403/59033261-920fdc00-8868-11e9-8061-8d0a51d93069.PNG)
fk_dataowner and fk_provider have changed from red to green (there has never been a value in this fields).

These red, yellow or green signs behind the fields are not ok:
![multiedit1](https://user-images.githubusercontent.com/1257403/59033668-7f49d700-8869-11e9-992c-cfe777482fae.PNG)
 
E.g. clear_heigth and material are the some (compare the labels) but have a red sign.

**Comment by urskaufmann** [comment]
I had then another not so cool effect: because I first thought its the last modification field, I added this field to the wastewater structure tab. After that, I could no more open the  attribute window - QGIS crashed everytime when I open the window until I removed this last mod field again.
![last-mod-error](https://user-images.githubusercontent.com/1257403/59033928-3d6d6080-886a-11e9-837f-8450dcf307b7.PNG)

The last mod field is already in the reach-tab. Is that a problem with triggers to update the last mod field?
